coordinator/src/services/translation/response/spine-response-handler.ts

```diff
@@ -542,3 +533,51 @@ export class ReleaseResponseHandler extends SpineResponseHandler<hl7V3.Prescript
     }
   }
 }
+
+export class ReleaseRejectionHandler extends SpineResponseHandler<hl7V3.PrescriptionReleaseRejectRoot> {
+  protected extractErrorCodes(
+    sendMessagePayload: hl7V3.SendMessagePayload<hl7V3.PrescriptionReleaseRejectRoot>
+  ): Array<hl7V3.Code<string>> {
+    const errorCode = sendMessagePayload.ControlActEvent.subject.PrescriptionReleaseReject
+      .pertinentInformation?.pertinentRejectionReason?.value
+    return errorCode ? [errorCode] : []
+  }
+
+  protected handleErrorResponse(
+    sendMessagePayload: hl7V3.SendMessagePayload<hl7V3.PrescriptionReleaseRejectRoot>,
+    logger: pino.Logger
+  ): TranslatedSpineResponse {
+    const spineResponse = super.handleErrorResponse(sendMessagePayload, logger)
+    const operationOutcome = spineResponse.fhirResponse as fhir.OperationOutcome
+    operationOutcome.issue.forEach((issue) => {
+      if (ReleaseRejectionHandler.issueNeedsDiagnosticInfo(issue)) {
+        issue.diagnostics = ReleaseRejectionHandler.getDiagnosticInfo(sendMessagePayload)
+      }
+      return issue
```

Comment:
not sure we need this return? its a forEach looping over issues and adding `diagnostics` if needed

if we were using a `.map` to build new issues we'd need the return but as is I think we're good without?